frontend/src/pages/notebookController/StartServerModal.tsx

```diff
@@ -64,10 +85,13 @@ const StartServerModal: React.FC<StartServerModalProps> = ({
       variant={ModalVariant.small}
       title="Starting server"
       isOpen={startShown}
-      showClose
+      showClose={
+        spawnStatus?.status === AlertVariant.danger || spawnStatus?.status === AlertVariant.warning
+      }
       onClose={() => (isNotebookRunning ? setStartModalShown(false) : onClose())}
```

Comment:
Maybe the close action should be:
- If we are in an error/warning state, `setStartModalShown(false)` // this is new state
- If we are still starting, `onClose()`
- If we are started, `setStartModalShown(false)`

Reply:
If we are in an error/warning state and we only `setStartModalShown(false)`, the error notebook will be still on the cluster but not running. We need to delete it if there is an error/warning.

Reply:
🤔 Yeah I guess if it's not spun up the code doesn't handle the deleting after the fact. Okay, nvm.